gradle/verification-metadata.xml

```diff
@@ -2866,6 +2866,11 @@
             <sha256 value="6e7ff132e341c0bd6b1046ca313fe8bee2187e0da68a272c487f60ebdfb425a8" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.lz4" name="lz4-java" version="1.7.1">
```

Comment:
Why do we have this now?

Reply:
Transitive dependency of flightrecorder: https://ge.gradle.org/s/5wkujgzpxlfvi/dependencies?dependencies=lz4&expandAll&focusedDependency=WzY5LDE3LDY1NjUsWzAsMCxbMF1dXQ&focusedDependencyView=versions